File: assignment/simpleinterest.cpp
#include<iostream>
using namespace std;
class Simple_Interest
{
float Principal,SimpleInterest;
int rate,time;
public:void Int()
{
    cout<<"Enter the Principal balance: \n";
    cin>>Principal;
    cout<<"Enter the Rate of Interest: \n";
    cin>>rate;
    cout<<"Enter the time in secs: \n";
    cin>>time;
    SimpleInterest=(Principal*time*rate)/100;
    cout<<"The Simple Interest is: \n"<<SimpleInterest;
}

};
int main()
{
    Simple_Interest obj;
    obj.Int();
}

File: Java Assignment 1/Simple_Interest.java
import java.util.*;
class Simple_Interest
{
float p,t,r,si;
void getdata()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter the Principal amount: ");
p=sc.nextFloat();
System.out.println("Enter the Time : ");
t=sc.nextFloat();
System.out.println("Enter  the rate: ");
r=sc.nextFloat();
}
void display()
{
si=((p*t*r)/100);
System.out.println("The Simple Interest is: "+si);
}
public static void main(String...args)
{
Simple_Interest obj=new Simple_Interest();
obj.getdata();
obj.display();
}
}